docs/reference/search/search-your-data/semantic-search-inference.asciidoc

```diff
@@ -103,8 +100,8 @@ POST _tasks/<task_id>/_cancel
 ==== Semantic search
 
 After the data set has been enriched with the embeddings, you can query the data using {ref}/knn-search.html#knn-semantic-search[semantic search].
-Pass a
-`query_vector_builder` to the k-nearest neighbor (kNN) vector search API, and provide the query text and the model you have used to create the embeddings.
+In case of dense vector models, pass a `query_vector_builder` to the k-nearest neighbor (kNN) vector search API, and provide the query text and the model you have used to create the embeddings.
+In case of a sparse vector model like ELSER, use a `text_expansion` query, and provide the query text with the model you have used to create the embeddings.
```

Comment:
Note: `text_expansion` has been deprecated as of 8.15, and we should guide people to use `sparse_vector` queries instead.